daphne/src/lib.rs

```diff
@@ -429,16 +445,113 @@ pub struct DapTaskConfig {
 
     /// The VDAF configuration for this task.
     pub vdaf: VdafConfig,
+}
+
+impl DapTaskParameters {
+    /// Construct a new task config using the taskprov extension. Return the task ID, the taskprov
+    /// advertisement (if applicable), and the payload of the report extension.
+    pub fn to_config_with_taskprov(
```

Comment:
this method seems to only be used in tests, that mean it can actually be moved into a test block in this file so it's out of the way of the main code. Other test modules in the library can still access it
```rs
#[cfg(test)]
mod test {
  impl DapTaskParameters {
    // ...
  }
}
```

Reply:
It's also needed by `daphne_worker_test`. I added `#[cfg(any(test, feature = "test-utils"))]` and left it where it is. We could move it to `testing` if that makes sense?